pages/faq.rst

```diff
@@ -118,12 +118,12 @@ Yup, we’re all over this too with read/write roles and group permissions.  To
 Do we have a user audit log for compliance?
 -------------------------------------------
 
-`Graylog Enterprise <https://www.graylog.org/enterprise>`_ includes an audit log plugin. You can explore the :ref:`documentation <auditlog_toc>` for more details.
+`Graylog Enterprise <https://www.graylog.org/enterprise>`_ includes a audit log functionality. You can explore the :ref:`documentation <auditlog_toc>` for more details.
```

Comment:
remove the a between includes and audit. Should read Graylog Enterprise includes audit log functionality.

Reply:
Will do, thank you! 👍